### PR TITLE
Task/11 all PO pending display first

### PR DESCRIPTION
**Pull request by mllrdev-jerry-bayoneta**

### CODE REVIEW

File: src/core/interfaces/purchase-order.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import PurchaseStatus from '@/core/enums/status.enum'
+import PurchaseAction from '@/core/enums/action.enum'
+interface IPurchaseOrder {
+    purchaseOrderNumber: string,
```
Comment (mllrdev-carlo-medel) [inline src/core/interfaces/purchase-order.ts]:
```suggestion
    id: string,
```

File: src/core/interfaces/purchase-order.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    description: string,
+    date: Date,
+    status: PurchaseStatus.APPROVED | PurchaseStatus.PENDING | PurchaseStatus.DECLINED | PurchaseStatus.CLOSE,
```
Comment (mllrdev-carlo-medel) [inline src/core/interfaces/purchase-order.ts]:
```suggestion
    status: PurchseStatus
```

File: src/core/component/OrderList.vue [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    const purchaseOrderSequences = computed(() => {
+      return [...props.orders].sort((a: IPurchaseOrder ) => {
+        if(a.status === props.status){
+          return -1
+        }
+        if(a.status > props.status){
+          return 1
+        }
+        return 0
+      })
```
Comment (mllrdev-carlo-medel) [inline src/core/component/OrderList.vue]:
```suggestion
      return [...props.orders].sort((a: IPurchaseOrder ) => {
        if (a.status === props.status) {
          return -1
        }
        
        if (a.status > props.status) {
          return 1
        }
        
        return 0
      })
```

File: src/views/Home.vue [hunk 4]
```diff
@@ -1,68 +1,132 @@
...
   <ion-page>
-    <ion-header :translucent="true">
-      <ion-toolbar>
-        <ion-title>Blank</ion-title>
-      </ion-toolbar>
-    </ion-header>
-    
+  <ion-toolbar>
+    <ion-button id="signInButton" button @click="buttonClickSignIn()">{{user.firstName}}</ion-button>
+    <h1>LOGO</h1>
```
Comment (mllrdev-carlo-medel) [inline src/views/Home.vue]: If this is an image, use an image tag and just use a placeholder for that image for now.

File: src/views/Home.vue [hunk 5]
```diff
@@ -1,68 +1,132 @@
...
 
 <script lang="ts">
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import PurchaseAction from '@/core/enums/action.enum';
+import PurchaseStatus from '@/core/enums/status.enum';
+import { IonContent, IonPage, IonTitle, IonToolbar, IonGrid, IonRow, IonButton } from '@ionic/vue';
+import { defineComponent, ref } from 'vue';
+import IPurchaseOrder from '../core/interfaces/purchase-order';
+import IUser from '../core/interfaces/user';
```
Comment (mllrdev-carlo-medel) [inline src/views/Home.vue]: Use complete path to avoid this scenario ../../../../core/interfaces

File: src/views/Home.vue (hunk 4 above)
Comment (mllrdev-jerry-bayoneta) [reply src/views/Home.vue]: done

File: src/core/component/OrderList.vue (hunk 3 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/component/OrderList.vue]: done

File: src/core/interfaces/purchase-order.ts (hunk 1 above)
Comment (mllrdev-jerry-bayoneta) [reply src/core/interfaces/purchase-order.ts]: done

File: src/views/Home.vue (hunk 5 above)
Comment (mllrdev-jerry-bayoneta) [reply src/views/Home.vue]: done

**Review by mllrdev-carlo-medel — APPROVED** [review]

**Pull request merged**